bigchaindb/commands/bigchain.py

```diff
@@ -86,6 +86,16 @@ def run_configure(args, skip_if_exists=False):
     conf['keypair']['private'], conf['keypair']['public'] = \
         crypto.generate_key_pair()
 
+    # select the correct config defaults based on the backend
+    print('Generating default configuration for backend {}'
+          .format(args.backend))
+    database = {}
```

Comment:
what about

```python
database = bigchaindb._database_map[args.backend]
```

It should not raise `KeyError`s because the `add_argument` has defined `choices`.